### PR TITLE
Corrected a few typos

### PR DESCRIPTION
**Pull request by Sayan98**

### CODE REVIEW

**Comment by jcadduono** [comment]
you missed some :)

**Comment by Sayan98** [comment]
@jcadduono most are fixed now! 👍

File: IDEAS_Bumble-Bee_OwnPush.md [hunk 1]
```diff
@@ -36,17 +36,17 @@ https://ownpush.com/
...
 * Background knowledge of android service (and their management)
-* Python developent experiance
+* Python development experience
 * Knowledge of  basic web protocols (HTTP(s) / Websockets / TCP-IP)
 
 
-## Optinal skills (You will learn during the project)
+## Optional skills (You will learn during the project)
 
 * Use of GIT base source management systems
```
Comment (jcadduono) [inline IDEAS_Bumble-Bee_OwnPush.md]: based

File: IDEAS_Bumble-Bee_OwnPush.md [hunk 2]
```diff
@@ -5,24 +5,24 @@ Develop and expand the OwnPush open source push messaging system, this includes
...
-to create a more open source friendly service for the entire developement community to make use of. Some
+to create a more open source friendly service for the entire development community to make use of. Some
 example outcomes would be.
 
 
-1. Publish an open source OwnPush server based on the exsisting documentation / examples
+1. Publish an open source OwnPush server based on the existing documentation / examples
 2. Upgrade the current OwnPush development process to help developers make use of the service
 3. Publish an enhanced version of the OwnPush android service.
 4. Publish a new  OwnPush client for a new operating system / platform
```
Comment (jcadduono) [inline IDEAS_Bumble-Bee_OwnPush.md]: double spaces

File: IDEAS_Bumble-Bee_OwnPush.md [hunk 3]
```diff
@@ -5,24 +5,24 @@ Develop and expand the OwnPush open source push messaging system, this includes
...
 Due to the nature of OwnPush there are a  multitude of expected results, all of which expands the OwnPush 
-to create a more open source friendly service for the entire developement community to make use of. Some
+to create a more open source friendly service for the entire development community to make use of. Some
 example outcomes would be.
 
 
-1. Publish an open source OwnPush server based on the exsisting documentation / examples
+1. Publish an open source OwnPush server based on the existing documentation / examples
 2. Upgrade the current OwnPush development process to help developers make use of the service
 3. Publish an enhanced version of the OwnPush android service.
```
Comment (jcadduono) [inline IDEAS_Bumble-Bee_OwnPush.md]: non-sentence ending with period

File: IDEAS_Bumble-Bee_OwnPush.md [hunk 4]
```diff
@@ -5,24 +5,24 @@ Develop and expand the OwnPush open source push messaging system, this includes
...
 
 OwnPush is an open source targeted push messaging system, currently only partially open source and is in
-need of extra developent to make it a fully open source tool for the entire mobile community. The majority
+need of extra development to make it a fully open source tool for the entire mobile community. The majority
 of OwnPush is split between a python web service and an android background service.
 
 
 ## Expected Results
 
 Due to the nature of OwnPush there are a  multitude of expected results, all of which expands the OwnPush
```
Comment (jcadduono) [inline IDEAS_Bumble-Bee_OwnPush.md]: double space, "the OwnPush to", expand

File: IDEAS_Bumble-Bee_OwnPush.md [hunk 5]
```diff
@@ -5,24 +5,24 @@ Develop and expand the OwnPush open source push messaging system, this includes
 * Creating an open source push server
 * Further developing the Ownpush android service / client
```
Comment (jcadduono) [inline IDEAS_Bumble-Bee_OwnPush.md]: inconsistent capitalization

File: IDEAS_Bumble-Bee_OwnPush.md [hunk 6]
```diff
@@ -36,17 +36,17 @@ https://ownpush.com/
...
 
-This project will requier a basic understanding of the following
+This project will require a basic understanding of the following
 
-* Android java developent
+* Android java development
 * Background knowledge of android service (and their management)
-* Python developent experiance
+* Python development experience
 * Knowledge of  basic web protocols (HTTP(s) / Websockets / TCP-IP)
```
Comment (jcadduono) [inline IDEAS_Bumble-Bee_OwnPush.md]: double space

File: IDEAS_Bumble-Bee_OwnPush.md [hunk 7]
```diff
@@ -36,17 +36,17 @@ https://ownpush.com/
...
 
-## Requierd Skills
+## Required Skills
 
-This project will requier a basic understanding of the following
+This project will require a basic understanding of the following
 
-* Android java developent
+* Android java development
 * Background knowledge of android service (and their management)
```
Comment (jcadduono) [inline IDEAS_Bumble-Bee_OwnPush.md]: services

**Comment by Sayan98** [comment]
🤷‍♂️

**Pull request closed without merging**